pkg/cmd/policy_enable.go

```diff
@@ -48,15 +53,69 @@ func newPolicyEnableCmd() *cobra.Command {
 				version = &cliArgs[1]
 			}
 
+			// Load the configuration from the user-specified JSON file into config object.
+			var config map[string]*json.RawMessage
+			if args.config != "" {
+				// config, err = loadJSONConfigFile(args.configFile)
```

Comment:
```suggestion

```

Reply:
lol... sorry. completely missed that. will make change.